crates/liboci-cli/src/checkpoint.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+use std::path::PathBuf;
+
+/// Checkpoint a running container
+#[derive(Parser, Debug)]
+pub struct Checkpoint {
+    #[clap(forbid_empty_values = true, required = true)]
+    pub container_id: String,
+    /// allow external unix sockets
```

Comment:
These comments also show up in the help for youki, so to be consistent with the other commands they should start with a capital letter.